packages/sdk/src/burn/burn.ts

```diff
@@ -1,23 +1,15 @@
-import { NearContractCall } from '../execute';
-import { GAS, ONE_YOCTO } from '../constants';
+import { DEPOSIT_FOR_BURN, GAS_FOR_BURN } from '../constants';
 import { mbjs } from '../config/config';
-import { TOKEN_METHOD_NAMES } from '../types';
+import { TOKEN_METHOD_NAMES, NearContractCall, BurnArgs, BurnReturnArgs } from '../types';
 import { ERROR_MESSAGES } from '../errorMessages';
 
-export type BurnArgs = {
-  contractAddress?: string;
-  tokenIds: string[];
-};
-
-export const DEPOSIT_FOR_BURN = ONE_YOCTO;
-export const GAS_FOR_BURN = GAS;
 
 /**
  * Burns one or more tokens from a given contract.
  * @param burnArguments {@link BurnArgs}
  * @returns contract call to be passed to @mintbase-js/sdk execute method
  */
-export const burn = ({ tokenIds, contractAddress = mbjs.keys.contractAddress }: BurnArgs): NearContractCall => {
+export const burn = ({ tokenIds, contractAddress = mbjs.keys.contractAddress }: BurnArgs): NearContractCall<BurnReturnArgs> => {
```

Comment:
Why are the other generics `<MethodArgs>`, but this `<MethodReturnArgs>`?

Reply:
Agree with feedback on "return args", as they are not really arguments to the function but a return "type", maybe something like `Response<Mint>`

Reply:
as explained in telegram, the naming was due that this is the typing for the args property inside NearContractCall, thats why I added Args in the end